sdk/provisioning/Azure.Provisioning.AppConfiguration/README.md

```diff
@@ -0,0 +1,81 @@
+# Azure Provisioning client library for .NET
+
+Azure.Provisioning.AppConfiguration is a cloud development kit that allows you to declaratively specify your Azure infrastructure natively in .NET.
+
+## Getting started
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.Provisioning.AppConfiguration --prerelease
+```
+
+### Prerequisites
+
+> You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/).
+
+### Authenticate the Client
+
+## Key concepts
+
+This library allows you to specify your infrastructure in a declarative style using dotnet.  You can then use azd to deploy your infrastructure to Azure diretly without needing to write or maintain bicep or arm templates.
+
+## Examples
```

Comment:
I filed https://github.com/Azure/azure-sdk-for-net/issues/43106 to track filling all these out properly